packages/@uppy/dashboard/src/index.js

```diff
@@ -535,6 +536,15 @@ module.exports = class Dashboard extends Plugin {
     //    To support IE.
     this.el.addEventListener('focus', this.recordIfFocusedOnUppyAtLeastOnce, true)
     this.el.addEventListener('click', this.recordIfFocusedOnUppyAtLeastOnce, true)
+
+    if (this.opts.inline) {
+      document.addEventListener('keydown', this.handleKeyDownInInline)
+    }
```

Comment:
Nitpicking: would `this.el.addEventListener` work?

Reply:
_Kind of_.
With `this.el`, in Link overlay: tabbing from the page requires double tab.
With `document`, in Link overlay: tabbing from page MISSES the Submit button.

Changed it to `this.el` for now, but this can be considered a #TODO for feature ammendments.